lib/ansible/executor/task_executor.py

```diff
@@ -832,12 +831,18 @@ def _get_connection(self, variables, templar):
                     if isinstance(i, string_types) and i.startswith("ansible_") and i.endswith("_interpreter"):
                         variables[i] = delegated_vars[i]
 
-        conn_type = self._play_context.connection
+        # load become plugin if needed
```

Comment:
Would it be worthwhile to add a default of no-op become plugin to always have a plugin to avoid the if/elses?
(Could possibly make things like plugins.connection.ConnectionBase.load_become simpler or possibly unneeded)

Reply:
I had been thinking about that, a noop plugin is easy enough, the current 'runas' is 99% that.